src/qt/coincontroldialog.cpp

```diff
@@ -231,7 +231,8 @@ void CoinControlDialog::copyAddress()
 // context menu action: copy transaction id
 void CoinControlDialog::copyTransactionHash()
```

Comment:
Method should be renamed as well as comment above, as it no more copies just transaction hash id alone.

Reply:
Done :)